zubhub_frontend/zubhub/src/views/add_remove_team_project/AddRemoveTeamProject.jsx

```diff
@@ -0,0 +1,159 @@
+import React, { useEffect, useState, useMemo, useCallback } from 'react';
+import { Grid, makeStyles, Backdrop } from '@material-ui/core';
+import styles from '../../assets/js/styles';
+import { customStyles } from './styles';
+import Project from '../../components/project/Project';
+import CheckCircleIcon from '@material-ui/icons/CheckCircle';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import ZubhubAPI from '../../api/api';
+import CustomButton from '../../components/button/Button';
+import { toast } from 'react-toastify';
+import clsx from 'clsx';
+
+/**
+ * @function AddTeamProject
+ * @author  Gift <giftnamulika@gmail.com>
+ *
+ * @todo - describe method's signature
+ */
+function AddTeamProject(props) {
+  const API = new ZubhubAPI();
+
+  console.log(props);
```

Comment:
you forgot this here. I know we have this in several places in the codebases right now but I think it's time we start thinking about cleaning things up a bit